src/translators/crate.py

```diff
@@ -425,7 +425,9 @@ def _get_select_clause(self, attr_names, aggr_method, aggr_period):
     def _get_limit(self, limit, last_n):
         # https://crate.io/docs/crate/reference/en/latest/general/dql/selects.html#limits
         default_limit = 10000
-
+        if os.environ.get("DEFAULT_LIMIT"):
```

Comment:
it'd be great if you used our [config reader](https://github.com/smartsdk/ngsi-timeseries-api/blob/master/src/utils/cfgreader.py) instead of plain python `os.env.get`. an example of how to use it is [over here](https://github.com/smartsdk/ngsi-timeseries-api/blob/77ed420c0a7532bcc13d941c0402f457cc40407a/src/translators/timescale.py#L81). I suppose we should use an IntVar, similar to the [reading of the Postgress port variable](https://github.com/smartsdk/ngsi-timeseries-api/blob/77ed420c0a7532bcc13d941c0402f457cc40407a/src/translators/timescale.py#L84)